includes/wikia/Extensions.php

```diff
@@ -796,7 +796,6 @@
 
 // Enable new style forums (/wiki/Special:Forum)
 if ( !empty( $wgEnableForumExt ) ) {
-	$wgArchiveWikiForums = true;
```

Comment:
If `$wgArchiveWikiForums` is not set in WF, we should still disable wiki style forums.

Reply:
good point!

Reply:
so If I got it correctly, `ArchiveWikiForums` should by default be set to true, right?

Reply:
Nope, that would disable wiki forums for wikis without the threaded forums extension :P I suggest setting the variable to default `null` instead. Then, this block, we can check the value and only set it to true if the value is `null`, which indicates that it was not overridden in WF.